Updated Dutch translation

Vistaus opened the pull request:
:-)

maoschanz commented:
no need to translate the todos lol

concerning the revealjs themes names, i didn't add comments for context and you translated quite literally, FYI the context is this: ![](https://cdn-images-1.medium.com/max/2000/1*ap5v9NJodgzByZEJzMCGfA.jpeg) i don't know if you want to change some of them?

Vistaus commented:
> no need to translate the todos lol

You're right lol, I removed the translations for those strings.

> concerning the revealjs themes names, i didn't add comments for context and you translated quite literally, FYI the context is this: ![](https://camo.githubusercontent.com/2d0bf1de0dfbbf3d20cd95fca93d31380025d081/68747470733a2f2f63646e2d696d616765732d312e6d656469756d2e636f6d2f6d61782f323030302f312a61703576394e4a6f64677a42795a454a7a4d434766412e6a706567) i don't know if you want to change some of them?

Didn't know it was about themes (I'm not familiar with revealjs). I changed a few. Thanks for the heads-up! :)
